src/goose/defaults.clj

```diff
@@ -11,12 +11,16 @@
 (def ^:no-doc in-progress-queue-prefix "goose/in-progress-jobs:")
 (def ^:no-doc process-prefix "goose/processes:")
 (def ^:no-doc heartbeat-prefix "goose/heartbeat:")
+(def ^:no-doc batch-prefix "goose/batch:")
 
 (def default-queue "default")
 (def schedule-queue "scheduled-jobs")
 (def dead-queue "dead-jobs")
 (def cron-queue "cron-schedules")
 (def cron-entries "cron-entries")
+(def enqueued-job-set "enqueued")
```

Comment:
suggestion(non-blocking): what do you think of having <state>-batch-set instead of <state>-job-set ?

Reply:
its actually a set of jobs for a batch.